Assets/Plugins/Windows/Editor/EOSOnPostprocessBuild_Windows.cs

```diff
@@ -304,11 +304,24 @@ public void OnPostprocessBuild(BuildReport report)
 
             InstallFiles(report);
             
-            string pathToEOSBootStrapperTool = GetPathToEOSBin() + "/EOSBootstrapperTool.exe";
-            string pathToEOSBootStrapper = GetPathToEOSBin() + "/EOSBootStrapper.exe";
-            string pathToEACIntegrityTool = GetPathToEOSBin() + "/EAC/anticheat_integritytool.exe";
+            string pathToEOSBootStrapperTool = Path.Combine(GetPathToEOSBin(), "EOSBootstrapperTool.exe");
+            string pathToEOSBootStrapper = Path.Combine(GetPathToEOSBin(), "EOSBootStrapper.exe");
+            string pathToEACIntegrityTool = Path.Combine(GetPathToEOSBin(), "EAC/anticheat_integritytool.exe");
```

Comment:
This should maybe be `Path.Combine(GetPathToEOSBin(), "EAC", "anticheat_integritytool.exe")`